### PR TITLE
AG-825 ADT: Reconfigure TMT proteomics to use new source files

### PR DESCRIPTION
**Pull request by j-hendrickson-sage**
Update to test_config.yaml with all changes to config.yaml; code changes in transform.py

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-- destination: &dest syn12177492
```
Comment (JessterB) [inline config.yaml]: We don't want to change this value; config.yaml should write to Agora Live Data, not to Agora Testing Data. Please revert this change (in config.yaml only).

File: config.yaml [hunk 2]
```diff
@@ -114,14 +112,16 @@
           - syn12514826
           - syn12514912
           - syn18689335
-          - syn11914808
+          - syn35221005
           - syn27211942
+        # column_rename:
+        #   GeneName: hgnc_gene_id
```
Comment (JessterB) [inline config.yaml]: If this column_rename isn't needed, let's remove it. If it is needed, let's uncomment it.

File: config.yaml [hunk 3]
```diff
@@ -114,14 +112,16 @@
...
         destination: *dest
 
     - gene_info:
         files:
           - name: gene_metadata
-            id: syn27644951
+            id: syn27572743
```
Comment (JessterB) [inline config.yaml]: We don't want to use the Agora Testing Data version of gene_metadata here, we want the Agora Live Data version, so please revert this change (in config.yaml only)

File: config.yaml [hunk 4]
```diff
@@ -31,15 +31,15 @@
...
         files:
           - name: agora_proteomics_tmt
-            id: syn32188234
+            id: syn35221005
             format: csv
         final_format: json
         provenance:
-          - syn32188234
+          - syn35221005
         column_rename:
```
Comment (JessterB) [inline config.yaml:43]: The new source file syn35221005 does not need most of these column_renames. 

Example:
syn35221005 doesn't have a peptide column, but it does have a uniqid column (remember that all colnames are automatically lowercased by the framework), so we don't need the `peptide: uniqid` column_rename.

Please reconcile the column names in syn35221005 with the list of column_rename and remove the ones we don't need. The same changes should be applied to test_config.yaml.

File: config.yaml [hunk 5]
```diff
@@ -270,11 +270,13 @@
...
+            id: syn35221005
+            format: csv
         final_format: json
         custom_transformations: 1
         provenance:
           - syn18689335
-          - syn33344657
+          - syn35221005
+        column_rename:
+          log_2fc: log2_fc
```
Comment (JessterB) [inline config.yaml]: Neither of the two source files has a `log_2fc` column now, they both have `log2_fc`, so we no longer need this rename. This change applies to both config files.

File: test_config.yaml [hunk 6]
```diff
@@ -35,11 +35,11 @@
...
         files:
           - name: agora_proteomics_tmt
-            id: syn32188234
+            id: syn35221005
             format: csv
         final_format: json
         provenance:
-          - syn32188234
+          - syn35221005
         column_rename:
```
Comment (JessterB) [inline test_config.yaml:43]: see comment on config.yaml here

File: test_config.yaml [hunk 7]
```diff
@@ -114,8 +112,10 @@
           - syn12514826
           - syn12514912
           - syn18689335
-          - syn11914808
+          - syn35221005
           - syn27211942
+        # column_rename:
+        #   GeneName: hgnc_gene_id
```
Comment (JessterB) [inline test_config.yaml]: see comment on config.yaml here

File: test_config.yaml [hunk 8]
```diff
@@ -270,13 +270,13 @@
...
+            id: syn35221005
+            format: csv
         final_format: json
         custom_transformations: 1
         provenance:
           - syn18689335
-          - syn32210527
+          - syn35221005
         column_rename:
           log_2fc: log2_fc
```
Comment (JessterB) [inline test_config.yaml]: see comment on config.yaml here

**Review by JessterB — CHANGES_REQUESTED** [review]

File: test_config.yaml (hunk 6 above)
Comment (j-hendrickson-sage) [reply test_config.yaml:43]: This doesn't need to be altered for the test_config.yaml, right?

Comment (j-hendrickson-sage) [reply test_config.yaml:43]: nevermind

File: config.yaml [hunk 9]
```diff
@@ -31,27 +31,21 @@
...
+          - syn35221005
         column_rename:
-          peptide: uniqid
-          geneid: hgnc_symbol
           protid: uniprotid
           ci_l: ci_lwr
           ci_h: ci_upr
-          ensg: ensemble_gene_id
           coefficient: log2_fc
           fdr_pval: cor_pval
```
Comment (JessterB) [inline config.yaml]: These column renames are still not quite right, we need to rename only hgnc and ensg:

        column_rename:
          genename: hgnc_symbol
          ensg: ensemble_gene_id

File: test_config.yaml [hunk 10]
```diff
@@ -35,23 +35,17 @@
...
+          - syn35221005
         column_rename:
-          peptide: uniqid
-          geneid: hgnc_symbol
           protid: uniprotid
           ci_l: ci_lwr
           ci_h: ci_upr
-          ensg: ensemble_gene_id
           coefficient: log2_fc
           fdr_pval: cor_pval
```
Comment (JessterB) [inline test_config.yaml]: These column renames are still not quite right, we need to rename only hgnc and ensg:

        column_rename:
          genename: hgnc_symbol
          ensg: ensemble_gene_id

**Review by JessterB — CHANGES_REQUESTED** [review]

**Review by JessterB — APPROVED** [review]
